src/main/java/org/opensearch/knn/plugin/stats/KNNStats.java

```diff
@@ -98,6 +100,28 @@ private void addQueryStats(ImmutableMap.Builder<String, KNNStat<?>> builder) {
 
     }
 
+    private void addMinScoreQueryStats(ImmutableMap.Builder<String, KNNStat<?>> builder) {
+        builder.put(
+            StatNames.MIN_SCORE_QUERY_REQUESTS.getName(),
+            new KNNStat<>(false, new KNNCounterSupplier(KNNCounter.MIN_SCORE_QUERY_REQUESTS))
+        )
+            .put(
+                StatNames.MIN_SCORE_QUERY_WITH_FILTER_REQUESTS.getName(),
+                new KNNStat<>(false, new KNNCounterSupplier(KNNCounter.MIN_SCORE_QUERY_WITH_FILTER_REQUESTS))
+            );
+    }
+
+    private void addMaxDistanceQueryStats(ImmutableMap.Builder<String, KNNStat<?>> builder) {
+        builder.put(
+            StatNames.MAX_DISTANCE_QUERY_REQUESTS.getName(),
+            new KNNStat<>(false, new KNNCounterSupplier(KNNCounter.MAX_DISTANCE_QUERY_REQUESTS))
+        )
+            .put(
+                StatNames.MAX_DISTANCE_QUERY_WITH_FILTER_REQUESTS.getName(),
+                new KNNStat<>(false, new KNNCounterSupplier(KNNCounter.MAX_DISTANCE_QUERY_WITH_FILTER_REQUESTS))
+            );
+    }
```

Comment:
can we merge these 2 functions with `addKNNQueryStats` and keep the same name which was `addQueryStats`

Reply:
ack